bangazonapi/models/product.py

```diff
@@ -58,12 +58,19 @@ def average_rating(self):
             number -- The average rating for the product
         """
         ratings = ProductRating.objects.filter(product=self)
-        total_rating = 0
-        for rating in ratings:
-            total_rating += rating.rating
 
-        avg = total_rating / len(ratings)
-        return avg
+        if len(ratings):
+            #sum of all the product ratings
+            total_rating = 0
+            for rating in ratings:
+                total_rating += rating.score
```

Comment:
If you look at the `ProductRating` model it's not going to be `.score` to get the integer value

Reply:
Thanks for taking a look. I just posted a question in the slack channel, so someone might pick that up and answer it. I see it's `.rating` according to the productrating.py file (which I honestly hadn't noticed in the models until now), but productrating.py has a field of `score` on the ERD and matches the `rating.py` model. Kind of confusing!